### DISCUSSION TITLE
Request: Adding Issues to Enhance External Contributions

### DISCUSSION THREAD

**Discussion by rokerzfirst101** [gofr-dev/gofr#415] [Q&A] [answered]
As an outsider interested in potentially contributing, I wanted to suggest adding open issues to the repository. This would provide a clearer roadmap of specific areas where project maintainers would welcome help and would make it much easier for external contributors like myself to get started.

**Comment by vikash** [comment, marked as answer]
Thanks for initiating this. We should be doing this with highest priority.
